scipy/signal/tests/test_bsplines.py

```diff
@@ -0,0 +1,214 @@
+# pylint: disable=missing-docstring
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy import array, nan
+from numpy.testing import (TestCase, assert_raises, run_module_suite,
+                           assert_allclose, assert_array_equal,
+                           assert_equal, assert_almost_equal)
+
+import scipy.signal.bsplines as bsp
+
+class TestBSplines(TestCase):
```

Comment:
Two newlines before class definition. Can you run `flake8` on this file and see if there are any other violations? Not all SciPy files conform but we try to make new additions abide by standard style if possible.